CriticalMapsMocksForTests/MockThemeController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SnapshotTester.swift
+//  CriticalMapsSnapshotTests
+//
+//  Created by MAXIM TSVETKOV on 05.10.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+@testable import CriticalMaps
+import SnapshotTesting
```

Comment:
Do we need to import SnapshotTesing? It is currently unavailable for the unit test target